components/monitoring/base/configure-prometheus.yaml

```diff
@@ -0,0 +1,223 @@
+---
```

Comment:
this file will be removed from this PR once https://github.com/redhat-appstudio/infra-deployments/pull/849 is merged (but I need it for now to test the connection to Prometheus)